packages/eslint-config-base/rules/imports.js

```diff
@@ -63,21 +63,14 @@ module.exports = {
           'test.{js,jsx}', // repos with a single test file
           'test-*.{js,jsx}', // repos with multiple top-level test files
           '**/*{.,_}{test,spec}.{js,jsx}', // tests where the extension or filename suffix denotes that it is a test
-          '**/jest.config.{js,ts}', // jest config
-          '**/jest.setup.{js,ts,mjs,cjs}', // jest setup
-          '**/vitest.setup.{js,ts,mjs,cjs}', // Vitest setup
-          '**/vitest.config.{js,ts,mjs,cjs}', // Vitest config
-          '**/vue.config.js', // vue-cli config
-          '**/webpack.config.{js,ts,mjs,cjs}', // webpack config
-          '**/webpack.config.*.{js,ts,mjs,cjs}', // webpack config
-          '**/rollup.config.js', // rollup config
-          '**/rollup.config.*.js', // rollup config
-          '**/gulpfile.js', // gulp config
-          '**/gulpfile.*.js', // gulp config
-          '**/Gruntfile{,.js}', // grunt config
-          '**/protractor.conf.js', // protractor config
-          '**/protractor.conf.*.js', // protractor config
-          '**/karma.conf.js', // karma config
+          '**/jest.config.{js,ts,mjs,cjs,mts,cts}', // jest config
```

Comment:
These all support TS extension?

Reply:
For jest yeah ts-jest does. I applied the same extensions to all these since it is a common standard

Reply:
Roger. I'm good with this for now.